exporter/prometheusremotewriteexporter/exporter_test.go

```diff
@@ -685,6 +686,16 @@ func Test_validateAndSanitizeExternalLabels(t *testing.T) {
 			map[string]string{"__key1_key__": "val1"},
 			false,
 		},
+		{"labels_that_start_with_underscore",
+			map[string]string{"_key_": "val1"},
+			map[string]string{"_key_": "val1"},
+			false,
+		},
+		{"labels_that_start_with_digit",
```

Comment:
Is there a test to ensure that users with sanitization turned off don't have their labels modified?

Reply:
@codeboten updated the testcases. please review.